NachoSpigot-Server/src/main/java/net/minecraft/server/MinecraftServer.java

```diff
@@ -834,7 +836,26 @@ public void B() {
                     }
                 }
             }
+        } */
+        for (i = 0; i < this.worlds.size(); ++i) { // Nacho
+            final WorldServer world = this.worlds.get(i); // Nacho
+            final boolean doDaylight = world.getGameRules().getBoolean("doDaylightCycle");
+            final long dayTime = world.getDayTime();
+            long worldTime = world.getTime();
+            final PacketPlayOutUpdateTime worldPacket = new PacketPlayOutUpdateTime(worldTime, dayTime, doDaylight);
+            for (EntityHuman entityhuman : world.players) {
+                if (!(entityhuman instanceof EntityPlayer) || (ticks + entityhuman.getId()) % 20 != 0) {
+                    continue;
+                }
+                EntityPlayer entityplayer = (EntityPlayer) entityhuman;
+                long playerTime = entityplayer.getPlayerTime();
+                PacketPlayOutUpdateTime packet = (playerTime == dayTime) ? worldPacket :
+                    new PacketPlayOutUpdateTime(worldTime, playerTime, doDaylight);
+                entityplayer.playerConnection.sendPacket(packet); // Add support for per player time
+            }
         }
```

Comment:
This optimization isn't worth it honestly, Because I believe it just gets buffered at the netty level and the calcs are low cost

Reply:
@HeathLoganCampbell Should we keep it or not?

Reply:
I can't see why this change was made in the first place, I would just revert this